### DISCUSSION TITLE
remap & Lua

### DISCUSSION THREAD

**Discussion by Lempossible** [vectordotdev/vector#17661] [Q&A]
I want to use Lua in Transform/Remap. Is it possible?  And how?

**Comment by StephenWakely** [comment]
You can't use Lua in the Remap transform. We have a separate [Lua transform](https://vector.dev/docs/reference/configuration/transforms/lua/) for this. 

There is nothing stopping you from having both transforms in your pipeline if necessary - try to get as much as possible done within the Remap transform since Vrl will perform better than Lua for most use cases.
